src/core/Loggyslav.ts

```diff
@@ -50,13 +52,13 @@ export class Loggyslav {
         this.classLoggers.push(classLogger);
     }
 
-    private setLogDataConfiguration(logDataConfiguration: TargetsConfiguration) {
+    private setTargetsConfiguration(logDataConfiguration: TargetsConfiguration) {
```

Comment:
`logDataConfiguration` could be renamed to `targetsConfiguration` to be consistent